# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/src/xpcpublic.h
# Commit: 955256297d6d
# Full Hash: 955256297d6dc83d52eb77a202c9c320622c0d13
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2019-11-05 16:16:12
# Regressor Bug: 1579367
# File Overlap Count: 1
# Description:
#   Bug 1579367 - Initialize XPCJSContext explicitly, after loading user prefs. r=kmag
#   
#   This way we get the correct values for start-up prefs in the parent process.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D51061
# ==============================================================================

diff -r 68e7fb2885b0 -r 955256297d6d js/xpconnect/src/xpcpublic.h
--- a/js/xpconnect/src/xpcpublic.h	Tue Nov 05 08:19:10 2019 +0200
+++ b/js/xpconnect/src/xpcpublic.h	Tue Nov 05 06:29:13 2019 +0000
@@ -710,6 +710,8 @@
   return sAutomationPrefIsSet && AreNonLocalConnectionsDisabled();
 }
 
+void InitializeJSContext();
+
 /**
  * Extract the native nsID object from a JS ID, IfaceID, ClassID, or ContractID
  * value.